crates/cheatnet/src/constants.rs

```diff
@@ -187,19 +192,25 @@ pub fn build_testing_state(predeployed_contracts: &Utf8PathBuf) -> DictStateRead
     let block_context = build_block_context();
     let test_account_class_hash = ClassHash(stark_felt!(TEST_ACCOUNT_CONTRACT_CLASS_HASH));
     let test_erc20_class_hash = ClassHash(stark_felt!(TEST_ERC20_CONTRACT_CLASS_HASH));
+    let test_contract_class_hash = ClassHash(stark_felt!(TEST_CONTRACT_CLASS_HASH));
 
     let class_hash_to_class = HashMap::from([
-        (test_account_class_hash, ContractClass::V1(account_class)),
+        (test_account_class_hash, ContractClass::V1(account_class.clone())),
+        // This is dummy put here only to satisfy blockifier 
+        // this class is not used and the test contract cannot be called
```

Comment:
Can we add a test which checks this?

Reply:
If you mean one checking if it cannot be called, such one already exists https://github.com/foundry-rs/starknet-foundry/blob/581ac740b5bf360f9df4b3fa63a83ece9ee564c6/crates/forge/tests/integration/test_state.rs#L336

Reply:
I updated the contract to be fully dummy as actually this changes made this test fail